community-roles.md

```diff
@@ -1,49 +1,46 @@
 
 ## Solid Core Contributor
-Solid Core Contributors work across projects on initiatives that enrich the
-Solid ecosystem as a whole. They must attend regular stand-ups and strategy
-meetings.
+Solid Core Contributors work across projects on initiatives that enrich the Solid ecosystem as a whole. Solid Core Contributors must attend the weekly recurring community support meeting.
 
 ## Project Core Contributor
-A Project Core Contributor is a key team member on an official Solid project.
-They may have issues assigned to them, review and merge pull requests, and must
-attend regular stand-ups and project strategy meetings.
-
-## Project Contributor
-A Project Contributor is anyone who contributes to an official Solid project.
-Contributions come in many forms, from coding and qa to experience design and
-technical content writing.
+A Project Core Contributor is a key team member on an official Solid project. Project Core Contributors must attend the weekly recurring community support meeting.
 
 ## Project Release Manager
-The Project Release Manager is responsible for determining which features and/or
-bug-fixes will make a given project release. They must attend regular stand-ups
-and project strategy meetings.
+The Project Release Managers are responsible for determining which features and/or bug-fixes will be merged into the release of their respective projects. Project release managers must attend the weekly recurring community support meeting.
+
+## Solid Coordinator
+Panel Coordinators are responsible for designing and running tests with the Solid Panelists as well as publishing test results to the Solid community. Panel Coordinators are expected to join the weekly user testing meeting which is moderated by the Community Manager.
+
+## Solid Panelist
+The Solid panel engages in continuous tests to improve the user experience. When a test is set up by the Solid panel coordinators the community manager will reach out to the Solid panel to ask if they would like to participate in this test. Tests will take approximately half an hour each on average. If the user tester is available and willing to take the test, they will be sent instructions by the Solid panel coordinators. Solid panelist can withdraw from the panel at any point. To volunteer to become a Solid panelist contact the community manager.
+
+## Solid Event Organiser
+The [Solid Event](solid-events.md) Organiser is responsible for organising and coordinatiing a Solid Event in a stated city. The position of Solid Event Organiser will be filled by an individual for as long as an event is scheduled in the future.
+
```

Comment:
Would like to propose a notion of community groups here, because I think they may provide some versatility. An example of some group ideas:
* Identity and Security
* Application Development
* Linked Data
* Solid Server
* Protocols and Standards
* User Experience
* Recognized Contributors (Alumni that may not be as active today but have played a substantial role and should be recognized)

Groups are a way to recognize people that are contributing around a given subject matter, which may span "projects". It may also be another angle on project contributors, where you back that out to a group which is responsible for a given set of projects / repositories. For example, the Identity and Security group could be focused on those projects / repos directly related to authentication, authorization, etc.

Reply:
Yes, I think we need to have a look at such divisions. One general goal I think we need to discuss is to what extent we would design the groups to be orthogonal, which I think would require us to put quite a lot of thought into it, or if we should just let them emerge and let it be quite fluid what goes where. There will be quite a lot of overlap between the "protocols and standards", "solid server" and "linked data", for example.